insns.def

```diff
@@ -1020,46 +1042,6 @@ branchnil
 /* for optimize                                           */
 /**********************************************************/
 
-/* push inline-cached value and go to dst if it is valid */
-DEFINE_INSN
-opt_getinlinecache
-(OFFSET dst, IC ic)
-()
-(VALUE val)
```

Comment:
Happy to see that you are able to completely eliminate the old form `opt_getinlinecache` 👍